openhands/sdk/preset/default.py

```diff
@@ -49,18 +44,11 @@ def get_default_agent_spec(
     llm: LLM,
     working_dir: str,
     cli_mode: bool = False,
-) -> AgentSpec:
-    agent_spec = AgentSpec(
+) -> Agent:
+    tool_specs = get_default_tools(working_dir=working_dir)
+    agent = Agent(
```

Comment:
@malhotra5 this should be helpful to you! We only load non-browser tools if cli_mode is true